scanpy/tests/test_highly_variable_genes.py

```diff
@@ -353,12 +355,13 @@ def test_compare_to_upstream(
     pbmc = pbmc68k_reduced()
     pbmc.X = pbmc.raw.X
     pbmc.var_names_make_unique()
-
     sc.pp.normalize_per_cell(pbmc, counts_per_cell_after=1e4)
     if func == "hvg":
         sc.pp.log1p(pbmc)
+        pbmc.X = array_type(pbmc.X)
```

Comment:
Too many differences propagate from the other functions before this.  In any case, I think this should really only test this individual change anyway.